src/core/loss_detection.c

```diff
@@ -422,6 +423,8 @@ QuicLossDetectionOnPacketSent(
         sizeof(QUIC_SENT_FRAME_METADATA) * TempSentPacket->FrameCount);
 
     LossDetection->LargestSentPacketNumber = TempSentPacket->PacketNumber;
+    CXPLAT_DBG_ASSERT(!SentPacket->Flags.IsDPLPMTUD ||
+                      (SentPacket->Flags.IsDPLPMTUD && !SentPacket->Flags.IsAckEliciting));
```

Comment:
Why should MTU probes **not** be ACK eliciting? That doesn't make sense. We need the ACK to get confirmation of the MTU.

Reply:
IsAckEliciting is actually a local flag which I think is misnamed. It actually means would resend. Because its only a local flag.

Reply:
Then we should rename it `IsRetransmittable`

Reply:
But I'm still not convinced I think it should still be ack eliciting, but the code should also use the probe flag when deciding to retransmit. For instance, if we do get an ACK for this, it should affect congestion control.

Reply:
These packets should affect the congestion window, but should not trigger congestion control. That is what happens currently.

Reply:
I realized that this assert was actually incorrect anyway. So removed.